README.md

```diff
@@ -121,15 +123,19 @@ smlbench/lookup-providers
 smlbench/store-get-value
 ```
 
-Then do
+Then run this comment to... (TODO: What does this do?)
 
 ```
 > TESTGROUND_BASEDIR=`pwd` testground -vv run dht/lookup-peers --builder=docker:go --runner=local:docker --build-cfg bypass_cache=true
 ...
 ```
 
+You should see... (TODO: What should I see?)
```

Comment:
You will see a bunch of logs that describe the steps of the test, from:

- Setting up the container
- Compilation of the test case inside the container
- Starting the containers (total of 50 as 50 is the default number of nodes for this test)
- You will see the logs that describe each node connecting to the others and executing a kademlia find-peers action.

Reply:
Do note that the Plans and Test cases are not final, so this might change meanwhile. We can keep updating it :)